<ns0:test-suite xmlns:ns0="urn:model.allure.qatools.yandex.ru" start="1527402566215" stop="1527402566266">
  <name>scripts.test001</name>
  <labels/>
  <test-cases>
    <test-case start="1527402566215" status="passed" stop="1527402566220">
      <name>Test_001.test_abc[1]</name>
      <attachments/>
      <labels>
        <label name="severity" value="normal"/>
        <label name="thread" value="2118-MainThread"/>
        <label name="host" value="dongyuedeMacBook-Pro.local"/>
        <label name="framework" value="pytest"/>
        <label name="language" value="cpython3"/>
      </labels>
      <steps/>
    </test-case>
    <test-case start="1527402566227" status="failed" stop="1527402566264">
      <name>Test_001.test_abc[2]</name>
      <failure>
        <message>AssertionError: assert 2 != 2</message>
        <stack-trace>self = &lt;scripts.test001.Test_001 object at 0x1046fe2b0&gt;, a = 2

    @pytest.mark.parametrize("a",[1,2,3])
    
    def test_abc(self,a):
&gt;       assert a!=2
E       assert 2 != 2

scripts/test001.py:7: AssertionError</stack-trace>
      </failure>
      <attachments/>
      <labels>
        <label name="severity" value="normal"/>
        <label name="thread" value="2118-MainThread"/>
        <label name="host" value="dongyuedeMacBook-Pro.local"/>
        <label name="framework" value="pytest"/>
        <label name="language" value="cpython3"/>
      </labels>
      <steps/>
    </test-case>
    <test-case start="1527402566265" status="passed" stop="1527402566266">
      <name>Test_001.test_abc[3]</name>
      <attachments/>
      <labels>
        <label name="severity" value="normal"/>
        <label name="thread" value="2118-MainThread"/>
        <label name="host" value="dongyuedeMacBook-Pro.local"/>
        <label name="framework" value="pytest"/>
        <label name="language" value="cpython3"/>
      </labels>
      <steps/>
    </test-case>
  </test-cases>
</ns0:test-suite>
